nix_update/__init__.py

```diff
@@ -26,6 +26,12 @@ def parse_args() -> Options:
     parser.add_argument(
         "--commit", action="store_true", help="Commit the updated package"
     )
+    parser.add_argument(
+        "-ve",
+        "--versionextractor",
+        help="Regex to extract version with",
```

Comment:
Some example would be nice:

```suggestion
        help="Regex to extract version with i.e. 'jq-(.*)'",
```

Reply:
Please also mention the flag in the README.